runtime/j9vm/jvm.c

```diff
@@ -450,6 +450,48 @@ static const J9SignalMapping signalMap[] = {
 #if defined(SIGPOLL)
 	J9_SIGNAL_MAP_ENTRY("POLL", SIGPOLL),
 #endif /* defined(SIGPOLL) */
+#if defined(SIGPWR)
+	J9_SIGNAL_MAP_ENTRY("CLD", SIGCLD),
+#endif /* defined(SIGPWR) */
```

Comment:
```suggestion
#endif /* defined(SIGCLD) */
```

Reply:
If it's supposed to be `SIGPWR, the two sets of ifdefs should be merged

Reply:
it should be `SIGCLD`; the MAP_ENTRY should be wrapped using the associated signal flag.